src/app/Fake.DotNet.NuGet/NuGet.fs

```diff
@@ -392,7 +404,12 @@ let NuGetPack setParams nuspecOrProjectFile =
 let NuGetPublish setParams =
     let parameters = NuGetDefaults() |> setParams
     use __ = Trace.traceTask "NuGet-Push" (packageFileName parameters)
-    publish parameters
+    try
+        publish parameters
+    with exn ->
+        if exn.InnerException <> null then exn.Message + "\r\n" + exn.InnerException.Message else exn.Message
+        |> replaceAccessKeys parameters
+        |> failwith
```

Comment:
Since https://github.com/fsprojects/Paket/pull/2423 I'm no longer a huge fan of `failwith` in a general `with` clause.
But it's not clear how we can remove the access keys in a secure way. They probably shouldn't be part of the error message in the first place if possible :/